client/my-sites/checkout/composite-checkout/components/wp-order-review-line-items.tsx

```diff
@@ -480,17 +481,27 @@ function returnModalCopyForProduct(
 ): ModalCopy {
 	const productType =
 		isPlan( product ) && hasDomainsInCart ? 'plan with dependencies' : product.product_slug;
-	return returnModalCopy( productType, translate, createUserAndSiteBeforeTransaction, isPwpoUser );
+	const isRenewal = isWpComProductRenewal( product );
+	return returnModalCopy( productType, translate, createUserAndSiteBeforeTransaction, isPwpoUser, isRenewal );
 }
 
 function returnModalCopy(
 	productType: string,
 	translate: ReturnType< typeof useTranslate >,
 	createUserAndSiteBeforeTransaction: boolean,
-	isPwpoUser: boolean
+	isPwpoUser: boolean,
+	isRenewal: boolean = false,
 ): ModalCopy {
 	switch ( productType ) {
 		case 'plan with dependencies': {
+			if ( isRenewal ) {
+				return {
+					title: String( translate( 'You are about to remove your plan renewal from the cart' ) ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **22** times:
`translate( 'You are about to remove your plan from the cart' )` *ES Score: 7*